app/models/snapshot/postcode_finder.rb

```diff
@@ -0,0 +1,36 @@
+require 'net/http'
+
+class Snapshot::PostcodeFinder
+  def country_postcodes(postcodes, country)
```

Comment:
It's not immediately obvious to me what gets returned by this. Could we change the name in some way to make it clearer?

Also, if PostcodeFinder is not tied to the snapshots, is it something that could go in lib to encourage reuse if we ever need to?